### ISSUE TITLE
Naps2 creates inaccessible pdf files

### ISSUE THREAD

**Issue by domasofan** [cyanfish/naps2#653]
**Describe the bug**

Naps2 seems not to create accessible pdf files.
At least there seems not to be a text layer in the pdf that screen readers could read.

**To Reproduce**

1. Scan a page with text
2. Save it as pdf

The pdf seems not to contain text that screenreaders can read.
As far as i knpow there was a bug in tesseract 4 which has been fixed in tesseract 5 that there was no text in the pdfs or at least it wasn't accessible to screen readers like nvda, orca, etc.

**Expected behavior**

pdfs should be readable with screen readers as well.
Blind people need ocr very often.

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Version: 8.2.0

**Additional context**

**Comment by cyanfish** [comment]
See #551
